Sources/Classes/RSContext.m

```diff
@@ -245,7 +245,7 @@ - (void) setSessionData:(RSUserSession *) userSession {
             [tempDict setObject:_externalIds forKey:@"externalId"];
         }
         if (_sessionId != nil) {
-            [tempDict setObject:_sessionId forKey:@"sessionId"];
+            [tempDict setObject:[NSNumber numberWithLong:_sessionId] forKey:@"sessionId"];
```

Comment:
Instead of `setObject`, you can use `setValue`; no need to convert into `NSNumber`